### PR TITLE
Restructure Non-Quickstart Docs

### PR DESCRIPTION
**Pull request by thomasheremans**
- Structure changes more lean
- Removing duplicate info on quickstarts
- Updating content to latest developments
- [Tabs for creator/leader choice](https://github.com/ObolNetwork/obol-docs/pull/86/commits/b4717500c3b48eb02648a3aab5573be624d71244)

Still needs to be updated
- Testnets
- Working Groups

### CODE REVIEW

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 1]
```diff
@@ -68,7 +85,24 @@ You will prepare the configuration file for the distributed key generation cerem
...
+      </TabItem>
+      <TabItem value="creator" label="Creator">
+    <ul><li>
+    
+    ⚠️ Turn the `Non-Operator` toggle ON.
+    
+    </li></ul>
+      </TabItem>
+    </Tabs>
+
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:105]: Doesn't let me comment on lines you haven't changed, but there's a typo 5 lines down from here on *transa**c**tion* fees. https://github.com/ObolNetwork/obol-docs/pull/86/files#diff-07d28d8844565e4e1aa5340dcf75bb2b9505ff18349f8b4440e9c6cabc2b3201R110

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 2]
```diff
@@ -68,7 +85,24 @@ You will prepare the configuration file for the distributed key generation cerem
 4. Follow the flow and accept the advisories.
 5. Configure the Cluster 
     - Input the `Cluster Name` & `Cluster Size` (i.e. number of operators in the cluster). The threshold for the cluster to operate sucessfully will update automatically.
-    - ⚠️ Leave the `Non-Operator` toggle off if you are a **Leader** and will participate in the cluster. Turn the toggle on if you are a **Creator** and won't participate.
+    
+    <Tabs groupId="leader-creator">
+      <TabItem value="leader" label="Leader" default>
+    <ul><li>
+    
+    ⚠️ Leave the `Non-Operator` toggle OFF.
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:93]:
```suggestion
    ⚠️ Leave the `Non-Operator` toggle <b>OFF</b>.
```

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 3]
```diff
@@ -68,7 +85,24 @@ You will prepare the configuration file for the distributed key generation cerem
...
+    
+    </li></ul>
+      </TabItem>
+      <TabItem value="creator" label="Creator">
+    <ul><li>
+    
+    ⚠️ Turn the `Non-Operator` toggle ON.
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:100]:
```suggestion
    ⚠️ Turn the `Non-Operator` toggle <b>ON</b>.
```

Comment (OisinKyne) [reply docs/int/quickstart/group/quickstart-group-leader-creator.md:100]: i do think we should have a rethink of that button, it's currently too much of a double negative imo

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 4]
```diff
@@ -79,11 +113,47 @@ You will prepare the configuration file for the distributed key generation cerem
...
-👉 If you are a **Leader**, you now have to follow the the [**Operator** Journey](./quickstart-group-operator). If you are a **Creator**, your journey ends here and you can monitor with the link whether the operators are confirming their agreement to the cluster by signing their approval.
\ No newline at end of file
+<Tabs groupId="leader-creator">
+  <TabItem value="leader" label="Leader" default>
+
+  👉 You now have to follow the the [**Operator** Journey](./quickstart-group-operator).
+  </TabItem>
+  <TabItem value="creator" label="Creator">
+
+   Your journey ends here and you can monitor with the link whether the operators are confirming their agreement to the cluster by signing their approval.
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:156]:
```suggestion
   Your journey ends here and you can monitor with the link whether the operators confirm their agreement to the cluster by signing their approval. Future versions of the launchpad will allow a creator to track a distributed validator's lifecycle in its entirety.
```

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 5]
```diff
@@ -79,11 +113,47 @@ You will prepare the configuration file for the distributed key generation cerem
...
 7. Share your cluster invite link with the operators. Following the link will show you a screen waiting for other operators to accept the configuration you created.
 
   ![Invite Operators](/img/Guide04.png)
 
-👉 If you are a **Leader**, you now have to follow the the [**Operator** Journey](./quickstart-group-operator). If you are a **Creator**, your journey ends here and you can monitor with the link whether the operators are confirming their agreement to the cluster by signing their approval.
\ No newline at end of file
+<Tabs groupId="leader-creator">
+  <TabItem value="leader" label="Leader" default>
+
+  👉 You now have to follow the the [**Operator** Journey](./quickstart-group-operator).
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:152]:
```suggestion
   👉 Once every participating operator has signed their approval to the terms, you will continue the [**Operator** journey](./quickstart-group-operator#step-3-run-the-dkg) by completing the distributed key generation step.
```

File: docs/int/quickstart/group/quickstart-group-operator.md [hunk 6]
```diff
@@ -73,7 +73,7 @@ After receiving the invite link created by the **Leader** or **Creator**, you wi
 
 ## Step 3. Run the DKG
 :::info
-For Charon v1, the [DKG](docs/dvk/01_distributed-validator-keys.md) needs to happen relatively synchronously between participants at an agreed time.
+For Charon v1, the [DKG](docs/dv/charon/dkg.md) needs to happen relatively synchronously between participants at an agreed time.
```
Comment (OisinKyne) [inline docs/int/quickstart/group/quickstart-group-operator.md:76]:
```suggestion
For the [DKG](docs/dv/charon/dkg.md) to complete, all operators need to be running the command simultaneously. It helps to coordinate an agreed upon time amongst operators at which to run the command.
```

**Review by OisinKyne — APPROVED** [review]
Good work. Couple places where I indicate that we should make tickets for more thorough overhauls of pages that are out of date, but out of scope of this PR. It would be great if you could make some of those tickets.

File: docs/cg/working-groups.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+- Jim McDonald
+- Prysmatic Labs
+- Sourav Das 
+- Mamy Ratsimbazafy 
+- Kobi Gurkan
+- Coinbase Cloud
+
+The intended universe of Phase 1 & Phase 1.5 will launch with no initial members, though they will immediately be available for submission by participants that have joined the Obol Proto community right [here](https://pwxy2mff03w.typeform.com/to/Kk0TfaYF). Everyone can join the proto community; however, working group participation will be based on relevance and skill set.
+
+
```
[the remark this answers is not in the document]
Comment (thomasheremans) [reply docs/cg/working-groups.md:146]: https://github.com/orgs/ObolNetwork/projects/7/views/1

File: docs/dv/testnet.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+Over the coming quarters, Obol Labs have and will continue to coordinate and host a number of progressively larger testnets to help harden the charon client and iterate on the key generation tooling.
+
+The following is a break down of the intended testnet roadmap, the features that are to be complete by each testnet, and their target start date and durations.
+
+# Testnets
+
+- [x] [Dev Net 1](#devnet-1)
+- [x] [Dev Net 2](#devnet-2)
+- [x] [Athena Public Testnet 1](#athena-public-testnet-1)
+- [ ] [Bia Attack net](#bia-attack-net)
```
[the remark this answers is not in the document]
Comment (thomasheremans) [reply docs/dv/testnet.md:19]: https://github.com/orgs/ObolNetwork/projects/7/views/1

**Pull request merged**